packages/base-styles/_variables.scss

```diff
@@ -21,6 +21,7 @@ $default-block-margin: 28px; // This value provides a consistent, contiguous spa
 $text-editor-font-size: 15px;
 $editor-line-height: 1.8;
 $mobile-text-min-font-size: 16px; // Any font size below 16px will cause Mobile Safari to "zoom in".
+$maximum-text-width: 50em;
```

Comment:
Is this variable likely to be reused in the future? I can't imagine it being used for anything other than data view descriptions at the moment, so I feel like it's ok to hard-code the value.

Reply:
I think it could be useful as more of the admin is redesigned (https://github.com/WordPress/gutenberg/issues/53322). But I'm fine to hard-code for now.

Reply:
Agreed with the above. Let's have a shared variable only:

- if we're sure it's needed across the whole project
- if we're sure that we can't reuse an existing one (even while updating it to the new value)